walkthroughs/eks/getting-started-with-appmesh/build-appserver-v2.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Build docker image
+APPSERVER_ECR_REPO=$(jq < cfn-output.json -r '.AppServerEcrRepo');
+docker build -t $APPSERVER_ECR_REPO ./yelb-appserver-v2/
+
+# Push image to ECR repo
+$(aws ecr get-login --no-include-email)
+docker push  $APPSERVER_ECR_REPO:latest
+
+# Generate Kubernetes deployment file
+cat > ./infrastructure/yelb_appserver_v2_deployment.yaml <<-EOF
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: yelb
+  name: yelb-appserver-v2
+  labels:
+    app: yelb-appserver-v2
+    tier: middletier
+spec:
+  type: ClusterIP
+  ports:
+    - port: 4567
+  selector:
+    app: yelb-appserver-v2
+    tier: middletier
+---
```

Comment:
Out of curiosity, why is this `cat`ed to a file? In general, it would be nice for all the k8s resources to be in existing YAMLs so folks can inspect them.

Reply:
Because I need to get the data from the ECR repository where the image was uploaded to when creating the file. All other files are there, but we are just creating it dinamically.

Reply:
I'd also recommend having the manifests in there and then `sed` the variable parts in. Is that doable @bemer?

Reply:
I'll change this in a bit.

Reply:
Changed!